src/storage-client/src/client.rs

```diff
@@ -124,6 +131,13 @@ pub struct RunIngestionCommand {
     pub description: IngestionDescription<CollectionMetadata>,
 }
 
+impl RunIngestionCommand {
+    /// Returns the IDs of all collections created by this command.
+    pub fn collection_ids(&self) -> impl Iterator<Item = GlobalId> + '_ {
+        std::iter::once(self.id).chain(self.description.source_exports.keys().copied())
```

Comment:
Hm, but we never issue `AllowCompaction` for these collections, right? That's the IDs I need, is there a good name for them? Otherwise I can just delete this method and inline it in the `CommandHistory`.

Reply:
I deleted the method and inlined its logic.